projects/common-components/src/lib/talks/talk-items-line/talk-items-line.component.scss

```diff
@@ -0,0 +1,28 @@
+.jsmu-talk-items-line {
+	position: relative;
+	top: 5px;
+	margin-bottom: 50px;
```

Comment:
BEM element sholdn't has margins
the parent container should be responsible for position

Reply:
done